aten/src/ATen/Context.h

```diff
@@ -88,24 +88,6 @@ class TORCH_API Context {
     }
   }
 
-  bool isPinnedPtr(
```

Comment:
`isPinnedPtr` is an API because of `class TORCH_API Context`. I think we should not remove it directly.

Reply:
From my point of view, it is reasonable to remove it now.

The api `isPinnedPtr` and `getPinnedMemoryAllocator` are just like `deviceCount`, we can call the function directly through `at::globalContext().getAcceleratorHooksInterface(device_type)`, instead of calling the wrapper function in Context, which makes Context more complicated.

@albanD , can you give some advices?

Reply:
From a google search, these don't seem used outside of core at all, so sounds ok to remove.
Note that https://github.com/search?q=+%2FglobalContext%5C%28%5C%29%5C.isPinnedPtr%2F&type=code&p=1 points at quite a few other places in the codebase though. Did you update these as well?